src/test/java/edu/cornell/kfs/module/ezra/service/ezraUpdateAwardImplTest.java

```diff
@@ -41,10 +41,17 @@ protected void setUp() throws Exception {
     public void testUpdateAwards () {
         Map fields = new HashMap();
         String awardProjectTitle = "ezraUpdateTest";
-        String awardProjectTitleAfter = ""; 
-        java.sql.Date today = SpringContext.getBean(DateTimeService.class).getCurrentSqlDate();
+        String awardProjectTitleAfter = "";
+        Date today = SpringContext.getBean(DateTimeService.class).getCurrentSqlDate();
```

Comment:
Can we create  a getDateTimeService() function, to support micro testing in the future?